### PR TITLE
Remote restarts and discord msg fix

### PR DESCRIPTION
**Pull request by skull132**
Fixes discord -> server PMs being shown to all staff.
Adds an instant restart command to the API which will nuke the process via TGS.
Fixes the remote announcement to use the same formatting as the usual announce.

### CODE REVIEW

File: code/modules/world_api/commands/server_query.dm [hunk 1]
```diff
@@ -110,7 +110,7 @@
 /datum/topic_command/get_count_mod/run_command(queryparams)
 	var/n = 0
 	for (var/client/client in clients)
-		if (client.holder && (client.holder.rights & R_MOD) && !(client.holder.rights & R_ADMIN))
+		if (client.holder && client.holder.rights == R_MOD)
```
Comment (Arrow768) [inline code/modules/world_api/commands/server_query.dm]: Mods have ban aswell. So that counts the trials

Comment (skull132) [reply code/modules/world_api/commands/server_query.dm]: Addressed.

File: code/modules/world_api/commands/server_query.dm [hunk 2]
```diff
@@ -110,7 +110,7 @@
 /datum/topic_command/get_count_mod/run_command(queryparams)
 	var/n = 0
 	for (var/client/client in clients)
-		if (client.holder && (client.holder.rights & R_MOD) && !(client.holder.rights & R_ADMIN))
+		if (client.holder && (client.holder.rights & R_MOD))
```
Comment (SoundScopes) [inline code/modules/world_api/commands/server_query.dm]: This will probably count HeadAdmin/Dev due to them having all perms

Comment (Sindorman) [reply code/modules/world_api/commands/server_query.dm]: That is true, it will account of Head Admin and Head Dev, I assume this is intentional?

**Comment by skull132** [comment]
YOU KNOW WHAT. Fuck these commands.

**Pull request merged**